src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -820,6 +820,13 @@ async def started(self):
             raise CallError('Automatically disabling LDAP service due to invalid configuration.',
                             errno.EINVAL)
 
+        """
+        Initialize state to "HEALTHY" until after booted.
+        """
+        if not await self.middleware.call('system.ready'):
+            await self.set_state(DSStatus['HEALTHY'])
```

Comment:
Wonder if HEALTHY is the right state? Do we have a JOINING state that would fit better?

Reply:
Yeah, that makes more sense.